src/containers/App.js

```diff
@@ -53,11 +53,11 @@ const App = () => {
     const uniqueTags = R.uniq(allTags).filter(Boolean);
 
     const stageMap = new Map();
-    R.flatten(mockData.map((comp) => {
+    mockData.forEach((comp) => {
       if(comp.stage !== undefined){
         return stageMap.set(comp.stageOrder, comp.stageName)
```

Comment:
if statement for comp.stage but this value never gets used. you only use stageorder and stagename